### PR TITLE
Jon branch

### PR DESCRIPTION
**Pull request by Jonathan169**

### CODE REVIEW

**Comment by sam-changtum** [comment]
@Jonathan169 , does Rich need to review?

**Comment by Jonathan169** [comment]
Rich is on vacation, i think its production ready as its just a form that i
made

**Comment by sam-changtum** [comment]
Got it!

**Comment by sam-changtum** [comment]
The create-account page is missing in the **dist**robution folder.

**Comment by Jonathan169** [comment]
my bad forgot to run the build, ill do it right now and create a pull
request

File: dist/signin.html [hunk 1]
```diff
@@ -0,0 +1,354 @@
+<!DOCTYPE html>
```
Comment (sam-changtum) [inline dist/signin.html]: Two things:
- Signin.html, an observation, is actually showing content for create-account.
- Missing required field indicators for create account content.

Other than that -- looks good to me.

Comment (Jonathan169) [reply dist/signin.html]: signin.html shouldnt be there can you check it again

**Pull request merged**